HACKING.md

```diff
@@ -125,6 +126,15 @@ make
 
 If you need help building this project for your platform, [contact us for help](https://www.kdab.com/about/contact/).
 
+## Online
+
+Instead of installing all dependencies and building Hotspot locally you may do so online with the following link:
```

Comment:
please add a sentence like

Note: This Gitpod setup is maintained externally by @GitMensch   and not part of the official KDAB hotspot repository.